src/OmniSharp.Http/OmniSharpHttpCommandLineApplication.cs

```diff
@@ -0,0 +1,21 @@
+using Microsoft.Extensions.CommandLineUtils;
+using OmniSharp.Internal;
+
+namespace OmniSharp.Http
+{
+    public class OmniSharpHttpCommandLineApplication : OmniSharpCommandLineApplication
```

Comment:
I'm going to keep complaining -- don't unnecessarily prefix class names with `OmniSharp`. Also, why is this public?